auction_jsligo/contracts/auction/main.jsligo

```diff
@@ -0,0 +1,257 @@
+#import "storage.jsligo" "Storage"
+#import "errors.jsligo" "Errors"
+#import "parameter.jsligo" "Parameter"
+#import "common.jsligo" "Common"
+#import "assert.jsligo" "Assert"
+
+export type storage = Storage.t
+export type parameter = Parameter.t
+export type return_ = [list<operation>, storage]
+
+
+let setCommissionFee = ([param, store] : [Parameter.set_fee_param, Storage.t]) : return_ => {
+    let _check_admin : unit = Assert.assert_only_admin(store);
```

Comment:
```suggestion
    Assert.assert_only_admin(store);
```
Also, should have been `Assert.onlyAdmin(store);` for better readability (and camelCase)

All further binds of function calls returning unit are not necessary